Ruby fails to install ronn package on CentOS 8.1

On CentOS 8.1 ruby complains about the Makefile's install script

```
$ make
mkdir -p ruby/bin; \
which ronn >& /dev/null && \
ln -sf `which ronn` ruby/bin/ronn || \
gem install ronn -i ruby
ERROR:  Use --install-dir or --user-install but not both
make: *** [Makefile:8620: ruby/bin/ronn] Error 1
```

Probably need to change "-i" to "--install-dir" in the make file command.

--- a/Makefile.am
+++ b/Makefile.am
@@ -794,7 +794,7 @@ ruby/bin/ronn:
 	mkdir -p ruby/bin; \
 	which ronn >& /dev/null && \
 	ln -sf `which ronn` ruby/bin/ronn || \
-	gem install ronn -i ruby
+	gem install ronn --no-user-install --install-dir ruby
 	if [ ! -f $@ ]; then pushd ruby/bin && ln -sf ronn.ruby* ronn && popd; fi
 
 $(ronn_man): man/%: ruby/bin/ronn ronn/%.ronn ronn/index.txt ronn/header.txt

